code/modules/antagonists/disease/disease_mob.dm

```diff
@@ -6,8 +6,8 @@ the new instance inside the host to be updated to the template's stats.
 */
 
 /mob/camera/disease
-	name = ""
-	real_name = ""
+	name = "Sentient Disease"
+	real_name = "Sentient Disease"
```

Comment:
I did this because having an empty name made the camera not appear in the right-click menu.